test/functional/apps/discover/group1/_shared_links.ts

```diff
@@ -73,19 +73,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       describe('permalink', function () {
         it('should allow for copying the snapshot URL', async function () {
-          const expectedUrl =
-            baseUrl +
-            '/app/discover?_t=1453775307251#' +
-            '/?_g=(filters:!(),refreshInterval:(pause:!t,value:60000),time' +
-            ":(from:'2015-09-19T06:31:44.000Z',to:'2015-09" +
-            "-23T18:31:44.000Z'))&_a=(columns:!(),filters:!(),index:'logstash-" +
-            "*',interval:auto,query:(language:kuery,query:'')" +
-            ",sort:!(!('@timestamp',desc)))";
+          const re = new RegExp(baseUrl + '/app/r\\?l=DISCOVER_APP_LOCATOR.+$');
```

Comment:
QQ: The URL generated by the app locator should stay the same, right? Would it make sense to actually test for the exact Url? So to be aware, if it breaks we should have a look into it, if the change was intended?

Reply:
:+1: Fixed in https://github.com/elastic/kibana/pull/154947/commits/63c7576a356ca279a4a452640dcb1bf0abb049e9